templates/github/.github/workflows/scripts/publish_client_gem.sh.j2

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+{% include 'header.j2' %}
+
+set -euv
+
+echo "---
+:rubygems_api_key: $RUBYGEMS_API_KEY" > ~/.gem/credentials
```

Comment:
Travis already comes with Ruby installed, Github Actions needs some extra steps for having this directory